keyboards/lily58/keymaps/nazu/config.h

```diff
@@ -0,0 +1,101 @@
+// #   define RGB_MATRIX_KEYRELEASES // reacts to keyreleases (instead of keypresses)
+// #   define RGB_DISABLE_AFTER_TIMEOUT 0 // number of ticks to wait until disabling effects
+#   define RGB_DISABLE_WHEN_USB_SUSPENDED // turn off effects when suspended
+// #   define RGB_MATRIX_LED_PROCESS_LIMIT (DRIVER_LED_TOTAL + 4) / 5 // limits the number of LEDs to process in an animation per task run (increases keyboard responsiveness)
+// #   define RGB_MATRIX_LED_FLUSH_LIMIT 16 // limits in milliseconds how frequently an animation will update the LEDs. 16 (16ms) is equivalent to limiting to 60fps (increases keyboard responsiveness)
+#   define RGB_MATRIX_MAXIMUM_BRIGHTNESS 150  // limits maximum brightness of LEDs to 200 out of 255. If not defined maximum brightness is set to 255
+#   define RGB_MATRIX_HUE_STEP 8
+#   define RGB_MATRIX_SAT_STEP 8
+#   define RGB_MATRIX_VAL_STEP 5
+#   define RGB_MATRIX_SPD_STEP 10
+// disabling effects
+#   define DISABLE_RGB_MATRIX_ALPHA_MODS
+#   define DISABLE_RGB_MATRIX_GRADIENT_UP_DOWN
+#   define DISABLE_RGB_MATRIX_GRADIENT_LEFT_RIGHT
+#   define DISABLE_RGB_MATRIX_BREATHING
+#   define DISABLE_RGB_MATRIX_BAND_SAT
+#   define DISABLE_RGB_MATRIX_BAND_VAL
+#   define DISABLE_RGB_MATRIX_BAND_PINWHEEL_SAT
+#   define DISABLE_RGB_MATRIX_BAND_PINWHEEL_VAL
+#   define DISABLE_RGB_MATRIX_BAND_SPIRAL_SAT
+#   define DISABLE_RGB_MATRIX_BAND_SPIRAL_VAL
+#   define DISABLE_RGB_MATRIX_CYCLE_ALL
+#   define DISABLE_RGB_MATRIX_CYCLE_LEFT_RIGHT
+#   define DISABLE_RGB_MATRIX_CYCLE_UP_DOWN
+#   define DISABLE_RGB_MATRIX_CYLCE_OUT_IN
+#   define DISABLE_RGB_MATRIX_CYLCE_OUT_IN_DUAL
+#   define DISABLE_RGB_MATRIX_CYLCE_PINWHEEL
+#   define DISABLE_RGB_MATRIX_CYLCE_SPIRAL
+#   define DISABLE_RGB_MATRIX_RAINBOW_MOVING_CHEVRON
+#   define DISABLE_RGB_MATRIX_DUAL_BEACON
+#   define DISABLE_RGB_MATRIX_RAINBOW_BEACON
+#   define DISABLE_RGB_MATRIX_RAINBOW_PINWHEEL
+#   define DISABLE_RGB_MATRIX_RAINDROPS
+#   define DISABLE_RGB_MATRIX_JELLYBEANS_RAINDROPS
+#   define DISABLE_RGB_MATRIX_HUE_BREATHING
+#   define DISABLE_RGB_MATRIX_HUE_PENDULUM
+#   define DISABLE_RGB_MATRIX_HUE_WAVE
+// #   define DISABLE_RGB_MATRIX_TYPING_HEATMAP
+#   define DISABLE_RGB_MATRIX_DIGITAL_RAIN
+#   define DISABLE_RGB_MATRIX_SOLID_REACTIVE_SIMPLE
+#   define DISABLE_RGB_MATRIX_SOLID_REACTIVE
+#   define DISABLE_RGB_MATRIX_SOLID_REACTIVE_WIDE
+#   define DISABLE_RGB_MATRIX_SOLID_REACTIVE_MULTIWIDE
+#   define DISABLE_RGB_MATRIX_SOLID_REACTIVE_CROSS
+#   define DISABLE_RGB_MATRIX_SOLID_REACTIVE_MULTICROSS
+#   define DISABLE_RGB_MATRIX_SOLID_REACTIVE_NEXUS
+#   define DISABLE_RGB_MATRIX_SOLID_REACTIVE_MULTINEXUS
+#   define DISABLE_RGB_MATRIX_SPLASH
+#   define DISABLE_RGB_MATRIX_MULTISPLASH
+#   define DISABLE_RGB_MATRIX_SOLID_SPLASH
+#   define DISABLE_RGB_MATRIX_SOLID_MULTISPLASH
+#   define RGB_MATRIX_TYPING_HEATMAP_DECREASE_DELAY_MS 50
+#endif
+
+#ifdef TAP_DANCE_ENABLE
+#   define TAPPING_TERM 200
+#endif
+
```

Comment:
This is the default.
```suggestion
```

Reply:
I put it into comments in case I need it again in the future. That was the sole reason I had it left in the code despite it being the default.